pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>idk.plugin.npc</groupId>
     <artifactId>NPC</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
```

Comment:
I mean why do we need "SNAPSHOT" in the version number at all?

Reply:
> I mean why do we need "SNAPSHOT" in the version number at all?

IDEA automatically set this up initially